### ISSUE TITLE
Multi-DB support

### ISSUE THREAD

**Issue by jsgao0** [Meesam/IssueTracker#3]
I think we may support more kinds of Database. And the syntaxes must be organized by each Database.

**Comment by Meesam** [comment]
yes we can,  i am planning to write logic for mongodb as well as mysql.

**Comment by jsgao0** [comment]
I mean the integration of it. Like creating another Linq.

**Comment by Meesam** [comment]
i think you are talking about create a module which can interact with all kind of database.
please explain.
